apps/src/templates/progress/ProgressLesson.jsx

```diff
@@ -206,7 +206,9 @@ class ProgressLesson extends React.Component {
         <div
           style={{
             ...styles.main,
-            ...((hiddenForStudents || isLockedForUser) && styles.translucent)
+            ...(((hiddenForStudents && !viewAs === ViewType.Teacher) ||
```

Comment:
For my education, is there a difference between checking `viewAs === ViewType.Student` and `viewAs !== ViewType.Teacher`?  Should we be using one or the other more consistently?

Reply:
My thinking was about signed out users but now that you bring it up I think signed out users act as students so we could probably use viewAs === ViewType.Student. I will update.